File: resources/source/java/innowake/components/source/cobol/HELLOW.java
package innowake.components.source.cobol;

import innowake.mee.base.api.*;
import innowake.mee.cobol.api.*;
import innowake.mee.cobol.api.cics.*;
import innowake.mee.cobol.api.Cobol.Transformation;

@Cobol(programID = "HELLOWORLD", version = Transformation.V3)
public class HELLOW extends CobolProgram {

    /* FD definitions */

    @Override
    public void run() {
        chainNextMethod("MAIN");
    }

    @Label
    void MAIN() {
        display("Hello, world.");
    }

}

File: resources/source/csharp/innowake/components/source/cobol/HELLOW.cs
using innowake.mee.@base.api;
using innowake.mee.cobol.api;
using innowake.mee.cobol.api.cics;
using InnoWake.Mee.Cobol.API.CICS;
using innowake.mee.cobol.api.adabas;
using InnoWake.Mee.Base.API;
using InnoWake.Mee.Cobol.API;

namespace Innowake.Components.Source.Cobol
{

    [Cobol]
    public partial class HELLOW : DotNetCobolProgram
    {
        public override void run()
        {
            chainNextMethod("MAIN");
        }

        [Label]
        void MAIN() {
            display("Hello, world.");
        }

    }
}
